build.gradle

```diff
@@ -367,6 +367,7 @@ def configureCluster(OpenSearchCluster cluster, Boolean securityEnabled) {
 integTest {
     useCluster testClusters.leaderCluster
     useCluster testClusters.followCluster
+    failFast = true
```

Comment:
Do we want this as part of the integ tests?

Reply:
Nope - this was just to get a successful run. Let me rebase and remove this.